src/daemon/control.rs

```diff
@@ -9,23 +9,17 @@
 use crate::{
     bitcoind::BitcoindError,
     database::{
-        actions::{
-            db_delete_spend, db_insert_spend, db_mark_broadcastable_spend, db_update_presigned_tx,
-            db_update_spend,
-        },
         interface::{
-            db_cancel_transaction, db_emer_transaction, db_list_spends, db_spend_transaction,
-            db_tip, db_unvault_emer_transaction, db_unvault_transaction, db_vault_by_deposit,
-            db_vault_by_unvault_txid, db_vaults, db_vaults_from_spend,
+            db_cancel_transaction, db_emer_transaction, db_unvault_emer_transaction,
```

Comment:
In commit 9ee70ef375f4282873c4dc4f0ffd5df759c75a3e, you should also remove the reference to the JSONRPC thread in main.rs, and i think there are elsewhere too